python/dev/.rat-excludes

```diff
@@ -0,0 +1,2 @@
+.rat-excludes
```

Comment:
related to [this old comment](https://github.com/apache/iceberg/pull/5840#discussion_r978666139) I'm curious whether it would now make sense to re-use the `check-license` script from the root dev folder

Reply:
I'd rather keep the two projects isolated so we have the possibility to split pyiceberg out in a separate repository.

Reply:
We can keep the two separate but use the same script right? I'd prefer that since it works well and is easy to verify.

Reply:
This will make the building more complicated. Before doing a `poetry build`, we need to copy the script from the parent directory to the Python directory.

Reply:
Why do we need to copy it? We could just commit a copy of it in place of the current script.